bundles/org.eclipse.cdt.lsp/META-INF/MANIFEST.MF

```diff
@@ -4,7 +4,8 @@ Bundle-Name: Language Server based C/C++ Editor
 Bundle-SymbolicName: org.eclipse.cdt.lsp;singleton:=true
 Bundle-Version: 1.0.0.qualifier
 Export-Package: org.eclipse.cdt.lsp,
```

Comment:
It could be reasonable to mark packages as `x-internal` until we have stable API

Reply:
I think this might be a good idea, however I won't change this in this PR, as I think this should be a separate discussion.

Reply:
Another PR for `x-internal` is fine with me